flink-core/pom.xml

```diff
@@ -33,6 +33,25 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<properties>
+		<surefire.module.config> <!--
+			required by JmxServer
+			-->--add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED <!--
+			PluginConfigTest
+			-->--add-opens=java.base/java.util=ALL-UNNAMED <!--
+			ExceptionUtilsTest
+			-->--add-opens=java.base/java.lang=ALL-UNNAMED <!--
+			-->--add-opens=java.base/java.lang.invoke=ALL-UNNAMED <!--
+			StateDescriptorTest
+			-->--add-opens=java.base/java.io=ALL-UNNAMED <!--
+			SerializersTest
+			-->--add-opens=java.base/java.net=ALL-UNNAMED <!--
```

Comment:
`java.net` doesn't occur in `SerializersTest` :thinking:

Reply:
It contains the Nested1 pojo containing a Flink `Path`, which internally contains a java.net.URI. This pojo is serialized via Kryo.